CHANGELOG-3.5.md

```diff
@@ -87,6 +87,7 @@ See [code changes](https://github.com/etcd-io/etcd/compare/v3.4.0...v3.5.0) and
 - [Embed Etcd does not override global/grpc logger](https://github.com/etcd-io/etcd/pull/12861) be default any longer. If desired, please call `embed.Config::SetupGlobalLoggers()` explicitly. 
 - [Embed Etcd custom logger should be configured using simpler builder `NewZapLoggerBuilder`](https://github.com/etcd-io/etcd/pull/12973).
 - Client errors of `context cancelled` or `context deadline exceeded` are exposed as `codes.Canceled` and `codes.DeadlineExceeded`, instead of `codes.Unknown`.
+- [ETCD_CLIENT_LOG_FORMAT env](https://github.com/etcd-io/etcd/pull/13339): Now supports log encoding (json, console). Only when set, overrides application-wide grpc logging settings
```

Comment:
This is 3.5 which is out, but we are releasing 3.6 next, so it needs to go here https://github.com/etcd-io/etcd/blob/main/CHANGELOG-3.6.md

Reply:
Yes, you are right. Let me change now.